esp/scm/ws_workunits_struct.ecm

```diff
@@ -882,3 +882,18 @@ ESPenum WUProtectFilter : string
     Protected("Protected"),
     NotProtected("NotProtected")
 };
+
+ESPStruct LogAccessFilter
```

Comment:
Seems like this covers everything. The target component(s) property (ComponentsFilter here) is the only thing I can recall from our last discussion that we'd said needed to be added. So looks good to me.

Reply:
Thanks @jeclrsg